http/oidc/src/test/java/org/wildfly/security/http/oidc/OidcProviderMetadataTest.java

```diff
@@ -0,0 +1,374 @@
+        assertTrue("http://localhost:8080/realms/myrealm/clients-registrations/openid-connect".equals(oidcProviderMetadata.getRegistrationEndpoint()));
+        assertTrue("http://localhost:8080/openid-connect".equals(withoutBooleansOidcProviderMetadata.getRegistrationEndpoint()));
+        assertNull(emptyOidcProviderMetadata.getRegistrationEndpoint());
+    }
+
+    @Test
+    public void testTokenEndpointAuthMethodsSupported() throws Exception {
+        List<String> l = oidcProviderMetadata.getTokenEndpointAuthMethodsSupported();
+        assertTrue(l.contains("client_secret_basic"));
+        assertNull(emptyOidcProviderMetadata.getTokenEndpointAuthMethodsSupported());
+    }
+
+    @Test
+    public void testTokenEndpointAuthSigningAlgValuesSupported() throws Exception {
+        List<String> l = oidcProviderMetadata.getTokenEndpointAuthSigningAlgValuesSupported();
+        assertTrue(l.contains("PS384"));
+        assertNull(emptyOidcProviderMetadata.getTokenEndpointAuthSigningAlgValuesSupported());
+    }
+
+    @Test
+    public void testClaimsSupported() throws Exception {
+        List<String> l = oidcProviderMetadata.getClaimsSupported();
+        assertTrue(l.contains("given_name"));
+        assertNull(emptyOidcProviderMetadata.getClaimsSupported());
+    }
+
+    @Test
+    public void testClaimTypesSupported() throws Exception {
+        List<String> l = oidcProviderMetadata.getClaimTypesSupported();
+        assertTrue(l.contains("normal"));
+        assertNull(emptyOidcProviderMetadata.getClaimTypesSupported());
+    }
+
+    @Test
+    public void testClaimsParameterSupported() throws Exception {
+        assertTrue(oidcProviderMetadata.getClaimsParameterSupported());
+        assertFalse(withoutBooleansOidcProviderMetadata.getClaimsParameterSupported());
+    }
+
+    @Test
+    public void testScopesSupported() throws Exception {
+        List<String> l = oidcProviderMetadata.getScopesSupported();
+        assertTrue(l.contains("offline_access"));
+        assertNull(emptyOidcProviderMetadata.getScopesSupported());
+    }
+
+    @Test
+    public void testRequestParameterSupported() throws Exception {
+        assertTrue(oidcProviderMetadata.getRequestParameterSupported());
+        assertFalse(withoutBooleansOidcProviderMetadata.getRequestParameterSupported());
+    }
+
+    @Test
+    public void testRequestUriParameterSupported() throws Exception {
+        assertTrue(oidcProviderMetadata.getRequestUriParameterSupported());
+        assertFalse(withoutBooleansOidcProviderMetadata.getRequestUriParameterSupported());
+    }
+
+    @Test
+    public void testPushedAuthorizationRequestEndpoint() throws Exception {
+        assertTrue("http://localhost:8080/realms/myrealm/protocol/openid-connect/ext/par/request".equals(oidcProviderMetadata.getPushedAuthorizationRequestEndpoint()));
+        assertNull(emptyOidcProviderMetadata.getPushedAuthorizationRequestEndpoint());
```

Comment:
I think this needs an assertNull for withoutBooleansProviderMetaData.

Reply:
done